pt-br/asymptoticnotation-pt.html.markdown

```diff
@@ -1,161 +0,0 @@
----
-category: Algorithms & Data Structures
-name: Asymptotic Notation
-contributors:
-    - ["Jake Prather", "http://github.com/JakeHP"]
-translators: 
-    - ["Carolina Knoll", "http://github.com/carolinaknoll"]
-lang: pt-br
----
```

Comment:
Are you sure this file is inferior or incorrect than another version?

Reply:
Yes, the two were translated by the same person. I deleted the old version and kept the new and corrected version.